### ISSUE TITLE
Could not connect to repository

### ISSUE THREAD

**Issue by Bulletbling** [SerpentDrago/skin.auramod#215]
Upon trying to open the addon repo "Auramod Repo AIO" it says "Could not connect to repository". 

Is it still up or broken?

**Comment by Goldenfreddy0703** [comment]
@Bulletbling sorry for not coming back to you in time but im not having any errors connecting to Auramod Repo AIO. This could be due to your connection or your provider and you may need to restart your router or something.

**Comment by Elara6331** [comment]
I am having the same issue. My internet works perfectly. Kodi works perfectly. Everything works except for that one repo.

**Comment by SerpentDrago** [comment]
Exactly what are you doing when you get that message? Exactly. And provide Kodi logs. Is this when you go to install from repo in Kodi or when adding The Kodi file source for The repo zip?

**Comment by Elara6331** [comment]
It happens when I click on "Auramod Repo AIO" after installing the zip. I will get logs if I can figure out how to retrieve them from my device (Apple TV 4, not Apple TV 4K).

**Comment by SerpentDrago** [comment]
so you add the filesource , you are able to "install from zip" the repo addon zip from that file source .  But when you go to "install from repo " and click the Auramod Repo thats now installed ... is when you get that error ? 

can you install from repo anything else ? any official kodi addons from the official kodi repo ? like say the official kodi log uploader :)

**Comment by Elara6331** [comment]
Yes, I can install from the official repos, and even other repos I've added.
